### ISSUE TITLE
`vergen-gix` does not consider the `matches` patterns when emitting describe output

### ISSUE THREAD

**Issue by SanchithHegde** [rustyhorde/vergen#363]
The `vergen-gix` crate does not consider the `matches` patterns specified when emitting the describe output. The `describe_match_pattern` field is just not being read anywhere in the [`vergen-gix/src/gix/mod.rs`](https://github.com/rustyhorde/vergen/blob/33fc3da860562126a3b38f1aec7cf5e5ac8767e9/vergen-gix/src/gix/mod.rs) file (it is being set in `describe()` though) as of opening this issue.

I couldn't find a way to pass the match patterns to `gitoxide` based on what I could search and based on the `gitoxide` docs. Let me know if I missed something, I can open a PR if there's a way to pass these patterns.

**Comment by nstinus** [comment]
I just stumbled upon this. Until this is supported, IMHO it would be better to remove `describe_match_pattern` to avoid any confusion.

**Comment by CraZySacX** [comment]
Yep this is true.  I left it in place to match the API pattern from `vergen-git2` and `vergen-cl` in the hopes `gix` would expose an easy way to do this in the interim, but that's still not the case.   I'll update the docs to reflect this, rather than removing the config for now.
